octis/evaluation_metrics/interpretability_metrics.py

```diff
@@ -0,0 +1,36 @@
+from octis.evaluation_metrics.metrics import AbstractMetric
+from gensim.corpora.dictionary import Dictionary
+from octis.evaluation_metrics.diversity_metrics import TopicDiversity
+from octis.evaluation_metrics.coherence_metrics import Coherence,_load_default_texts
+
+
+class TopicInterpretability(AbstractMetric):
+    def __init__(
+        self,  # noqa
+        texts: str = None,
+        topk: int = 10
+    ) -> None:
+        """
+        Initialize metric
+        Parameters
+        ----------
+        texts : list of documents (list of lists of strings)
+        topk : how many most likely words to consider in
+        the evaluation
+        measure : (default 'c_npmi') measure to use.
```

Comment:
this is not an input argument. We can just remote this line.

Reply:
Yes, sure!